app/Phragile/BurndownChart.php

```diff
@@ -38,26 +38,12 @@ private function calculatePointsClosedPerDay()
 		}
 	}
 
-	private function getClosedTaskIDs()
-	{
-		return array_map(function($task)
-		{
-			return $task['id'];
-		}, array_filter($this->tasks->getTasks(), function($task)
-			{
-				return $task['closed'];
-			})
-		);
-	}
-
 	private function getClosedTaskTimes()
 	{
-		$taskTransactions = $this->phabricator->getTaskTransactions($this->getClosedTaskIDs());
-
 		return array_map(function($transactions)
```

Comment:
Should the parameter of the anon function not be called `$transaction`? Or is the transcations field an array of arrays?